lib/src/components/NxSmallThreatCounter/__tests__/NxSmallThreatCounter.test.tsx

```diff
@@ -4,280 +4,157 @@
-      expect(severeCounter.find('.nx-small-threat-counter__category')).toHaveText('Severe');
-
-      const moderateCounter = component.find('.nx-small-threat-counter--moderate');
-      expect(moderateCounter.find('.nx-small-threat-counter__count')).toHaveText('44');
-      expect(moderateCounter.find('.nx-small-threat-counter__category')).toHaveText('Moderate');
-
-      const lowCounter = component.find('.nx-small-threat-counter--low');
-      expect(lowCounter.find('.nx-small-threat-counter__count')).toHaveText('33');
-      expect(lowCounter.find('.nx-small-threat-counter__category')).toHaveText('Low');
-
-      const noneCounter = component.find('.nx-small-threat-counter--none');
-      expect(noneCounter.find('.nx-small-threat-counter__count')).toHaveText('22');
-      expect(noneCounter.find('.nx-small-threat-counter__category')).toHaveText('None');
-
-      const unspecifiedCounter = component.find('.nx-small-threat-counter--unspecified');
-      expect(unspecifiedCounter.find('.nx-small-threat-counter__count')).toHaveText('42');
-      expect(unspecifiedCounter.find('.nx-small-threat-counter__category')).toHaveText('Unspecified');
-    });
-
-    it('does not cut off the displayed number', function() {
-      const component = getMountedComponent({
-        criticalCount: 34592384579138,
-        severeCount: 1
-      });
-
-      const criticalCounter = component.find('.nx-small-threat-counter--critical');
-      expect(criticalCounter.find('.nx-small-threat-counter__count')).toHaveText('34592384579138');
-      expect(criticalCounter.find('.nx-small-threat-counter__category')).toHaveText('Critical');
-
-      const severeCounter = component.find('.nx-small-threat-counter--severe');
-      expect(severeCounter.find('.nx-small-threat-counter__count')).toHaveText('1');
-      expect(severeCounter.find('.nx-small-threat-counter__category')).toHaveText('Severe');
+      unspecifiedCount: 11
     });
+    const criticalCounter = getCounter(component, 'critical'),
+        severeCounter = getCounter(component, 'severe'),
+        moderateCounter = getCounter(component, 'moderate'),
+        lowCounter = getCounter(component, 'low'),
+        noneCounter = getCounter(component, 'none'),
+        unspecifiedCounter = getCounter(component, 'unspecified');
+
+    expect(criticalCounter).toHaveTextContent('66');
+    expect(severeCounter).toHaveTextContent('55');
+    expect(moderateCounter).toHaveTextContent('44');
+    expect(lowCounter).toHaveTextContent('33');
+    expect(noneCounter).toHaveTextContent('22');
+    expect(unspecifiedCounter).toHaveTextContent('11');
+  });
+
+  it('displays maxDigits\' nines and a plus sign for any counts that go over the max when maxDigits is specified' +
+    'default is three digits', function() {
+    const twoDigitMaxComponent = quickRender({ criticalCount: 100000, maxDigits: 2 }),
+        fiveDigitDefaultComponent = quickRender({ moderateCount: 1000000 }),
```

Comment:
five?

Reply:
Corrected here 1134bd59d82384c4fa802b214beaf740d78e7307.

Reply:
> Corrected here https://github.com/sonatype/sonatype-react-shared-components/commit/1134bd59d82384c4fa802b214beaf740d78e7307.

That wasn't the change I was expecting.  Based on the variable name above this one, I thought the number in the variable name referred to the `maxDigits` value.  So shouldn't this one be `threeDigitDefaultComponent`?

Reply:
Oh yes, updated here 5dc2262b93ca132993d670e04a39072e35273ad6, sorry for the confusion.